src/looprestoration_tmpl_8.rs

```diff
@@ -1037,7 +1061,77 @@ unsafe extern "C" fn loop_restoration_dsp_init_x86(
         (*c).sgr[2] = Some(dav1d_sgr_filter_mix_8bpc_avx512icl);
     }
 }
-
+// This calculates things slightly differently than the reference version.
+// This version calculates roughly this:
+// int32_t sum = 0;
+// for (int i = 0; i < 7; i++)
+//     sum += mid[idx] * fv[i];
+// sum = (sum + rounding_off_v) >> round_bits_v;
+// This function assumes that the width is a multiple of 8.
```

Comment:
This comment is for `dav1d_wiener_filter_v`, not `wiener_filter_neon`.  I think it's fine to leave out for now; we'll add the comments back as we clean up, devirtualize, and genericize `looprestoration`.

Reply:
Good catch!